content/en/docs/reference/glossary/csi.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Container Storage Interface (CSI)
+title: CSI (Container Storage Interface)
```

Comment:
Please revert -- standard docs style is spelled out first, then acronym in parentheses (as in original), then acronym throughout the rest of the page/file.

Reply:
Reverted